src/pages/accountsettings/AccountSettings.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState, Component } from "react";
+import { Link } from "react-router-dom";
+// import "../signup/signup.css";
+
+import "../../common.css"
+
+const initialValues = {
+  email: "",
+  password: "",
+};
```

Comment:
Nice creating an object for the initial values of the state, but you don't use the variable.
Thus, it should be removed.